apps.yaml

```diff
@@ -106,6 +106,7 @@ qm9-schnet-uncertainty:
             A BioLib app for predicting atomization energies of QM9-like molecules with calibrated uncertainty using an ensemble of graph neural network models. For details about the method, please see our paper: https://doi.org/10.1088/2632-2153/ac3eb3.
         authors: Jonas Busk
         external_url: https://biolib.com/jbusk/qm9-schnet-uncertainty/
+        documentation_url: https://github.com/BIG-MAP/qm9-schnet-uncertainty-app
```

Comment:
I would point to the README.md file:
documentation_url: https://github.com/BIG-MAP/qm9-schnet-uncertainty-app/blob/main/README.md

I would also insert a key-value pair for git_url:
git_url: https://github.com/BIG-MAP/qm9-schnet-uncertainty-app/

Reply:
Dear author, 
Thank you for your pull request. I left a few comments that should be addressed before merging.

Reply:
Thank you for the comment. I have made the requested changes.